<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1233</width>
    <height>711</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="CurveImagePlot" native="true">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>30</y>
      <width>700</width>
      <height>300</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="CurveImageIIPlot" native="true">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>590</y>
      <width>481</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="TwoDImageIIPlot" native="true">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>650</y>
      <width>491</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="TwoDImagePlot" native="true">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>1</y>
      <width>649</width>
      <height>407</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="newtable">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>628</y>
      <width>641</width>
      <height>192</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>5</number>
    </property>
    <property name="columnCount">
     <number>4</number>
    </property>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QChartView" name="Historgram" native="true">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>350</y>
      <width>700</width>
      <height>300</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1233</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menuAnalysis">
    <property name="title">
     <string>Analysis</string>
    </property>
    <addaction name="actionViewer"/>
    <addaction name="actionSection"/>
    <addaction name="actionRoughness"/>
    <addaction name="actionPower_Spectral_Denstiy"/>
    <addaction name="actionFlatten"/>
    <addaction name="actionThree_Dimension"/>
    <addaction name="actionHistogram"/>
   </widget>
   <widget class="QMenu" name="menuFilters">
    <property name="title">
     <string>Filters</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuCommands">
    <property name="title">
     <string>Commands</string>
    </property>
    <addaction name="actionDouble_Channel"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHistory">
    <property name="title">
     <string>History</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAnalysis"/>
   <addaction name="menuFilters"/>
   <addaction name="menuCommands"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHistory"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSection"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/new/prefix1/image/image_open.ico</normaloff>:/new/prefix1/image/image_open.ico</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSection">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/new/prefix1/image/general.ico</normaloff>:/new/prefix1/image/general.ico</iconset>
   </property>
   <property name="text">
    <string>Section</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionRoughness">
   <property name="text">
    <string>Roughness</string>
   </property>
  </action>
  <action name="actionPower_Spectral_Denstiy">
   <property name="text">
    <string>Power Spectral Denstiy</string>
   </property>
  </action>
  <action name="actionDouble_Channel">
   <property name="text">
    <string>Double Channel</string>
   </property>
  </action>
  <action name="actionFlatten">
   <property name="text">
    <string>Flatten</string>
   </property>
  </action>
  <action name="actionThree_Dimension">
   <property name="text">
    <string>Three Dimension</string>
   </property>
  </action>
  <action name="actionViewer">
   <property name="text">
    <string>Viewer</string>
   </property>
  </action>
  <action name="actionHistogram">
   <property name="text">
    <string>Histogram</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header location="global">qchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
 <slots>
  <signal>signal1()</signal>
 </slots>
</ui>
